Kitodo/src/main/resources/checkstyle.xml

```diff
@@ -27,14 +23,8 @@
             <property name="message" value="Avoid using corresponding octal or Unicode escape."/>
         </module>
 
-        <!--<module name="AvoidEscapedUnicodeCharacters">
-            <property name="allowEscapesForControlCharacters" value="true"/>
-            <property name="allowByTailComment" value="true"/>
-            <property name="allowNonPrintableEscapes" value="true"/>
-        </module>-->
-
         <module name="LineLength">
-            <property name="max" value="120"/>
+            <property name="max" value="200"/>
```

Comment:
That's really long! Personally I'd prefer a much lower limit (typically 80 for other projects).